SU2_CFD/include/numerics_structure.hpp

```diff
@@ -66,13 +66,13 @@ class CNumerics {
   su2double *Enthalpy_formation;
   su2double Prandtl_Lam;        /*!< \brief Laminar Prandtl's number. */
   su2double Prandtl_Turb;    /*!< \brief Turbulent Prandtl's number. */
-  
```

Comment:
I just reverted the unnecessary whitespace changes.  I agree that they should be eliminated to reduce the number of "false conflicts," and I also agree that SU2 could benefit from more consistent use of whitespace.